releases/release_notes_09_2022.md

```diff
@@ -0,0 +1,282 @@
+# About 
+Welcome :-). 
+
+We are thrilled to share with you Confidential Containers (COCO) release X.X.X .
+
+In this document we will take you through the content of this release, installation instructions, deploying workloads and troubleshooting when things go wrong. 
+
+# Release notes 
+
+## Goals 
+
+This release focused on the following:
+
+- **Simplicity** - using the operator to deploy and configure
+- **Stability** - Supporting CI for the key workflows of the release
+- **Documentation** - details instruction of how to deploy and use this release
+
+## Use cases 
+
+This release supports the following use cases:
+
+- Creating a sample COCO workload
+- Creating a COCO workload using a pre-existing encrypted image
+- Creating a COCO workload using a pre-existing encrypted image on CC HW
+- Building a new encrypted container image and deploying it as a COCO workload
+
+## Limitations 
+
+The following are known limitations of this release: 
```

Comment:
Presumably this will be a list of brief descriptions with links to a GitHub URL for each. What about hardware support for the initial release? That might be better in a separate section but we should state clearly what hardware and architectures are known to work.

Reply:
I'm a bit late to the party, but I would have added a TBD here, so that we can make sure we "closed all the TBDs" by the end of the release.